packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -793,28 +789,34 @@ describeIf(process.platform !== 'win32' && !isMacOrWindowsCI)('MongoDB', () => {
     expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
-            // *** WARNING ***
-            // 
-            // The following fields had data stored in multiple types. Either use Json or normalize data to the wanted type.
-            // - Model "users", field: "numberOrString1", chosen data type: "Json"
-            // - Type "UsersHobbies", field: "numberOrString2", chosen data type: "Json"
-            // - Type "UsersHobbiesObjects", field: "numberOrString3", chosen data type: "Json"
-            // 
-        `)
+                        // *** WARNING ***
+                        // 
+                        // The following fields had data stored in multiple types. Either use Json or normalize data to the wanted type.
+                        // - Model "users", field: "numberOrString1", chosen data type: "Json"
+                        // - Type "UsersHobbies", field: "numberOrString2", chosen data type: "Json"
+                        // - Type "UsersHobbiesObjects", field: "numberOrString3", chosen data type: "Json"
+                        // 
+                `)
   })
 
   // describeIf is making eslint not happy about the names
   // eslint-disable-next-line jest/no-identical-title
   test('basic introspection --url', async () => {
     const introspect = new DbPull()
     const result = introspect.parse(['--print', '--url', MONGO_URI])
-    await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(`
-            Preview feature not enabled: MongoDB Introspection connector is a Preview feature and needs the \`mongoDb\` Preview feature flag. See https://www.prisma.io/docs/concepts/database-connectors/mongodb
-
-          `)
```

Comment:
Why were we testing _this_ in a test with a generic name?

Reply:
Tests were copy/pasted across providers, you can have many tests with the same name in jest :trophy: :trollface:

Reply:
Hm yeah, but then the snapshot should not have been merged but the test adapted to actually ahve the preview feature. This way the test was pretty useless for the actual functionality under test I would say.